src/components/Grid/_grid-settings.scss

```diff
@@ -0,0 +1,24 @@
+// ----------------------------------
+// UI Components
+// Grid
+// Settings
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+// TODO: #5906 discuss approach to grid/layout overall
+
+:root {
+  --grid-max-width: 1344px;
+}
+
+// 8px spacing unit based on (16px / 2)
+$grid-spacing-unit: 8px;
+
+// Gutter
+$grid-gutter-xs: $grid-spacing-unit * 1.5;
+$grid-gutter-sm: $grid-spacing-unit * 2.5;
+$grid-gutter-md: $grid-spacing-unit * 5;
+$grid-gutter-lg: $grid-gutter-md;
+$grid-gutter-xl: $grid-spacing-unit * 6;
```

Comment:
I wanted to convert all of these to CSS variables, however it's not possible to use Sass relational operators on them (as the `_grid-mixins.scss` file does, a lot)